src/sdam/topology.ts

```diff
@@ -469,9 +468,11 @@ export class Topology extends TypedEventEmitter<TopologyEvents> {
         readPreferenceServerSelector(readPreference),
```

Comment:
```js
    const selectServerOptions = {
      operationName: 'ping',
```

Can't comment on the line but since there's no ping anymore maybe this should be "handshake"?

Reply:
Updated.